gxaircom.py

```diff
@@ -0,0 +1,47 @@
+import arrow
+import requests
+
+from winds_mobi_provider import Provider, StationStatus, ureg, Q_, Pressure
+
+
+class GxAircom(Provider):
+    provider_code = "gxaircom"
+    provider_name = "gxaircom.com"
+
+    def process_data(self):
+        self.log.info("Processing GxAircom data...")
+        data = requests.get("http://www.mgenet.at/gxaircom/stations.php", timeout=(self.connect_timeout, self.read_timeout)).json()
+        for station in data:
+            winds_station = self.save_station(
+                provider_id=station["stationId"],
+                short_name=station["stationName"],
+                name=station["stationName"],  # Lets winds.mobi provide the full name with the help of Google Geocoding API
+                latitude=station["lat"],
+                longitude=station["lon"],
+                status=StationStatus.GREEN if station["online"] == 1 else StationStatus.RED,
+                altitude=station["alt"],
+            )
+
+            measure_key = arrow.get(station["DT"], "YYYY-MM-DD HH:mm:ss").int_timestamp
+            measures_collection = self.measures_collection(winds_station["_id"])
+
+            if not self.has_measure(measures_collection, measure_key):
+                new_measure = self.create_measure(
+                    for_station=winds_station,
+                    _id=measure_key,
+                    wind_direction=station["wDir"],
+                    wind_average=Q_(station["wSpeed"], ureg.kilometer / ureg.hour),
+                    wind_maximum=Q_(station["wGust"], ureg.kilometer / ureg.hour),
+                    temperature=Q_(station["temp"], ureg.degC) if station["temp"] is not None else None,
+                    pressure=Pressure(station["pressure"], qnh=None, qff=None) if station["pressure"] is not None else None,
+                )
+                self.insert_new_measures(measures_collection, winds_station, [new_measure])
+        self.log.info("...Done !")
+
```

Comment:
A provider should always catch the exceptions. I improved the provider example in the README with a generic exception handling:

https://github.com/winds-mobi/winds-mobi-providers/blob/main/README.md#add-a-new-provider-to-windsmobi